default/transforms.conf

```diff
@@ -127,7 +127,7 @@ REGEX  = Priority\:\s+([^\]]+)
 FORMAT = severity_id::$1
 
 [opnsense_suricata_signature]
-REGEX  = suricata\[[^\]]+\]\:\s+\[[^\]]+\]\s+([^\]]+)\s+\[
+REGEX  = suricata\[[^\]]+\]\:\s+\[[^\]]+\]\s+\[[^\]]+\]([^\]]+)\[
```

Comment:
Which version of suricata are you using? The proposed regex does not capture anything on my instances.

Reply:
I'm running;

OPNsense 20.1-amd64
FreeBSD 11.2-RELEASE-p16-HBSD
OpenSSL 1.1.1d 10 Sep 2019

in the package manager it shows

suricata | 4.1.6 | 6.03MiB | GPLv2

Reply:
Thank you. I seem to be on the same version as you. Can you provide me with a few more data points?
If you navigate to `Services > Intrusion Detection > Administration > (Settings Tab) > Right below the Setting tab, toggle the "Advanced Mode" switch`:
- Is the "Enable syslog alerts" box checked?
- Is the "Enable eve syslog output" box checked?
- Is the "Log package payload" box checked?

Reply:
Hi - info below;
    IPS mode = disabled (non blocking)
    Promiscuous mode = disabled
    Is the "Enable syslog alerts" box checked? - Yes
    Is the "Enable eve syslog output" box checked? - Yes
    Is the "Log package payload" box checked? - No

Example event below 
`Mar  3 16:24:02 OPNsense suricata[25620]: [wDrop] [1:2012647:5] ET POLICY Dropbox.com Offsite File Backup in Use [Classification: Potential Corporate Privacy Violation] [Priority: 1] {TCP} 162.125.136.1:443 -> 192.168.0.176:65289 `